molfeat/calc/base.py

```diff
@@ -30,7 +30,40 @@ def __init__(cls, name, bases, attrs):
 
 
 class SerializableCalculator(abc.ABC, metaclass=_CalculatorMeta):
-    """Interface to define a serializable calculator"""
+    """Interface to define a serializable calculator
+
+    ???+ tip "Subclassing SerializableCalculator"
```

Comment:
`???+ tip` typo or legit?

Reply:
legit. Learned that yesterday

Reply:
Nice looking

![image](https://user-images.githubusercontent.com/528003/230729303-250fad22-16f9-4ca3-8f10-3d985b811779.png)

Reply:
FYI: https://jejjohnson.github.io/research_journal/resources/mkdocs/#extensions